### ISSUE TITLE
Some issues during runtime

### ISSUE THREAD

**Issue by old0110** [lbaa2022/LLMTaskPlanning#6]
Hello author, I am running a program on a remote server (without a UI interface). How can I see the simulation environment on my local computer? I executed according to the code in the "Benchmarking on Watch Ad Help NL Using Headless PC" section

**Comment by old0110** [comment]
<img width="895" alt="微信图片_20240902225509" src="https://github.com/user-attachments/assets/e6864c1d-775e-4820-a0b7-ffc9a012b8e7">
There is another issue where false execution fails during execution. What is the reason for this? If I cannot see the execution process of the robot in the simulation environment, how can I solve it?

**Comment by youngwoo-yoon** [comment]
I don't know how to see simulator view in a headless server, but you can render scenes to video files by setting 'recording': True in `conf/config_wah.yaml` or `conf/config_wah_headless.yaml`. Please be aware that the results might be slightly different according to this option.

**Comment by youngwoo-yoon** [comment]
There are various reasons for the failure of low-level actions. I suggest you to debug the steps or inspect result videos for a specific test case.

**Comment by old0110** [comment]
> There are various reasons for the failure of low-level actions. I suggest you to debug the steps or inspect result videos for a specific test case.



> I don't know how to see simulator view in a headless server, but you can render scenes to video files by setting 'recording': True in `conf/config_wah.yaml` or `conf/config_wah_headless.yaml`. Please be aware that the results might be slightly different according to this option.

ok，thank you~

**Comment by old0110** [comment]
Hello author, after successfully running the program, I found that the success rate was 0. What is the situation

**Comment by youngwoo-yoon** [comment]
That is strange. We've ran the experiments on at least 3 different systems and there was no issue, and we're also using this code base actively on other projects. Could you check again and describe the problem more in detail?

**Comment by old0110** [comment]
> That is strange. We've ran the experiments on at least 3 different systems and there was no issue, and we're also using this code base actively on other projects. Could you check again and describe the problem more in detail? I recommend to use a local pc for testing.
![image](https://github.com/user-attachments/assets/ecc9c9c9-17e0-418e-af3c-cc12fe513ed4)
As shown in the picture, the overall success rate was 0, and the success rate of the subtasks was also very low. The model I used was EleutherAI/gpt-neo-1.3B

**Comment by youngwoo-yoon** [comment]
This is correct. As you can see in Figure 2 (b) in the paper, subgoal success rate is quite low with small-size models in WAH-NL tasks.

**Comment by old0110** [comment]
> This is correct. As you can see in Figure 2 (b) in the paper, subgoal success rate is quite low with small-size models in WAH-NL tasks.

Okay, I understand, but is it a bit strange that the overall success rate is only 0，hhhh
